src/main/java/com/adamk33n3r/runelite/watchdog/EventHandler.java

```diff
@@ -354,7 +354,8 @@ private void onWallObjectDespawned(WallObjectDespawned wallObjectDespawned) {
     private void onTileObjectSpawned(TileObject tileObject, SpawnedAlert.SpawnedDespawned mode, SpawnedAlert.SpawnedType type) {
         final ObjectComposition comp = this.client.getObjectDefinition(tileObject.getId());
         final ObjectComposition impostor = comp.getImpostorIds() != null ? comp.getImpostor() : comp;
-        this.onSpawned(impostor.getName(), mode, type);
+        String unformattedName = Text.removeFormattingTags(imposter.getName());
```

Comment:
I think it might be best if we move this to `onSpawned` so that it affects all spawnable objects and not just tile objects.

Reply:
You got it!